writeengine/wrapper/writeengine.cpp

```diff
@@ -2415,10 +2430,9 @@ int WriteEngineWrapper::insertColumnRecsBinary(const TxnID& txnid,
                     if (curFbo != lastFbo)
                     {
                         RETURN_ON_ERROR(AddLBIDtoList(txnid,
-                                                      lbids,
-                                                      colDataTypes,
```

Comment:
We should also remove the definitions of `lbids`, `colDataTypes` from this function.